tests/core/pyspec/eth2spec/test/helpers/random.py

```diff
@@ -0,0 +1,113 @@
+from random import Random
```

Comment:
also looks like this is just refactor of code

Reply:
yes! plus made `randomize_epoch_participation` be able to pick the epoch (prev or current)